english/marshmallow.md

```diff
@@ -94,4 +94,6 @@ Do you:
 
 [hide the lighter fork and rope in your pocket](hide/hide-chk.md)
 
-[make a necklace out of the marshmallows](necklace/necklace.md)
\ No newline at end of file
+[make a necklace out of the marshmallows](necklace/necklace.md)
+
+[light up the wall 2](light-up2/light-up2.md)
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.